src/containerapp/azext_containerapp/containerapp_job_decorator.py

```diff
@@ -184,11 +184,9 @@ def validate_arguments(self):
         validate_container_app_name(self.get_argument_name(), AppType.ContainerAppJob.name)
         validate_create(self.get_argument_registry_identity(), self.get_argument_registry_pass(), self.get_argument_registry_user(), self.get_argument_registry_server(), self.get_argument_no_wait())
         if self.get_argument_yaml() is None:
-            if self.get_argument_replica_timeout() is None:
-                raise RequiredArgumentMissingError('Usage error: --replica-timeout is required')
 
-            if self.get_argument_replica_retry_limit() is None:
-                raise RequiredArgumentMissingError('Usage error: --replica-retry-limit is required')
+            if self.get_argument_trigger_type() is None:
```

Comment:
Hi @Juancpani 
As I will remove `ContainerAppJobCreateDecorator `in extension and import it from Core: https://github.com/Azure/azure-cli/tree/dev/src/azure-cli/azure/cli/command_modules/containerapp.

I think it will be better to add validation for `if self.get_argument_trigger_type() is None:` in `ContainerAppJobPreviewCreateDecorator`, for example:
`ContainerAppPreviewCreateDecorator.validate_arguments`